addons/account_check_printing/models/account_journal.py

```diff
@@ -57,12 +57,13 @@ def _inverse_check_next_number(self):
                 journal.check_sequence_id.sudo().number_next_actual = int(journal.check_next_number)
                 journal.check_sequence_id.sudo().padding = len(journal.check_next_number)
 
-    @api.model
-    def create(self, vals):
-        rec = super(AccountJournal, self).create(vals)
-        if not rec.check_sequence_id:
-            rec._create_check_sequence()
-        return rec
+    @api.model_create_multi
+    def create(self, vals_list):
+        journals = super().create(vals_list)
+        for journal in journals:
+            if not journal.check_sequence_id:
+                journal._create_check_sequence()
```

Comment:
call in batch and update `_create_check_sequence` to avoid creating a sequence when it exists ?